### ISSUE TITLE
Standardize victory cues

### ISSUE THREAD

**Issue by hexagonrecursion** [wesnoth/wesnoth#5514]
### Game and System Information
Battle for Wesnoth v1.15.9+dev x86_64

### Describe the desired feature
The vast majority of the scenarios I have played so far have trained me to expect certain visual and auditory cues when I win a scenario. This includes:

* The music changes to victory.ogg or victory2.ogg
* There is a popup telling me that I won

In the absence of the cues it takes me a bit to realize that the scenario did in fact end. This inconsistency is surprising and confusing.

#### Examples (this is not an exhaustive list)
Tutorial: S1: no victory popup, no music change
AToTB: 04_Return_to_the_Village: no victory popup, no music change
TSG: 08a_Vengeance: no victory popup, no music change
TSG: 08b_The_Tides_of_War: no victory popup, no music change
Liberty: 07_Glory: no victory popup, no music change
HttT: 24_Battle_for_Wesnoth: no victory popup, no music change

**Comment by Pentarctagon** [comment]
I'm pretty sure that when a scenario ends that the screen gets a grey tint and the End Scenario button is shown?

**Comment by hexagonrecursion** [comment]
Victory cues for the vast majority of campaign scenarios:
* gray tint
* end scenario button
* victory music
* victory popup

But in some rare cases half the cues are missing. This inconsistency is confusing for me. I was genuinely confused when this happened at the end of TSG.

**Comment by Pentarctagon** [comment]
There are 73 cases in mainline campaigns where the victory music and popup aren't shown (aka have `carryover_report=no`), so I don't think it's as much of a convention as you're thinking it is.

Moreover, for your first example listed, the final message of the first tutorial scenario actually explains exactly what the screen being greyed out at the end means.

**Comment by hexagonrecursion** [comment]
> There are 73 cases in mainline campaigns where the victory music and popup aren't shown (aka have `carryover_report=no`), so I don't think it's as much of a convention as you're thinking it is.

I play the scenarios one at a time. I do not see the full picture. Humans are very good at extrapolation. I see the popup once -> I expect the popup every time. I hear the music change once -> I expect the music to change every time. Every time I see the popup and hear the music change the connection is reinforced.

**Comment by Pentarctagon** [comment]
I would expect that the screen being greyed out would be the strongest correlation, in that case.

**Comment by Wedge009** [comment]
I agree that one does get accustomed to the victory music pieces as an indication of the (successful) end of a scenario, but I also find the grey tint a clear indication too.

**Comment by cooljeanius** [comment]
There are some cases, though, where even though the result has to be called "victory" to move onto the next scenario, it doesn't really make sense as a "victory" in an emotional or narrative sense. Like, in AToTB S01, for example, playing the victory music right after discovering that Baran had been kidnapped would be major mood whiplash.
